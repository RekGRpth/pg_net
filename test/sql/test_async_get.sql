begin;

    select net.async_get('http://supabase.io');

    select
        id, url
    from
        net.request_queue;

rollback
