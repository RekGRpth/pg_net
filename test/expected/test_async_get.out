begin;
    select net.async_get('http://supabase.io');
 async_get 
-----------
         1
(1 row)

    select
        id, url
    from
        net.request_queue;
 id |        url         
----+--------------------
  1 | http://supabase.io
(1 row)

rollback
